content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -19,7 +19,7 @@ To learn more about the _motivation_ for Prisma, check out the [**Why Prisma?**]
 Prisma is an [open source](https://github.com/prisma/prisma) next-generation ORM. It consists of the following parts:
 
 - **Prisma Client**: Auto-generated and type-safe query builder for Node.js & TypeScript
-- **Prisma Migrate** (_experimental_): Declarative data modeling & migration system
+- **Prisma Migrate** (Preview): Declarative migration system
 - **Prisma Studio**: GUI to view and edit data in your database
 
 Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](../prisma-in-your-stack/rest), a [GraphQL API](../prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
```

Comment:
Not really _any_ I guess since we don't support all versions of Node? 
That's outside of scope though so fine to leave as is